assets/prototype/application/services/apollo/mutations/useEntityMutation.ts

```diff
@@ -45,7 +61,7 @@ const useEntityMutation = (type, id = '') => {
 		 */
 		const { onUpdate, ...mutationOptions } = mutator(mutationType, input);
 
-		let update;
+		let update: MutationUpdaterFn;
```

Comment:
This is really nice since we don't have to guess how this variable will be changed and this kind of typing will prevent accidental overwrite.